datarootsio/terraform-module-azure-datalake
# Run Terratests in CI

* Run Terratests as part of the CI pipeline

I made them a separate workflow so that we can keep the validation as required and this one as optional (because they might fail because of Azure capacity limitations)

* Add Go and Jetbrains to gitignore
* Remove Go installer step since Go is preinstalled by default
* Remove Terraform installer step since Terraform is installed by default

This step also has an optional wrapper that is enabled by default, but which should be disabled for Terratest.

* Fixed an issue (found with tests 🎉) with the shell scripts (they use bash-only syntax but were marked to use the default shell)
* Go module support
